src/android/patches/makeStringsPatch.js

```diff
@@ -0,0 +1,10 @@
+module.exports = function makeStringsPatch(params) {
+  const patch = Object.keys(params).map(key =>
+    `    <string moduleConfig="true" name="${key}">${params[key]}</string>`
```

Comment:
> I think we should escape characters here, as obviously I am not sure how my-awesome-key is going to be handled in a Java code (as a name)

Valid point. But I don't know any programming language that allows you to use dashes in the name. So I think we shouldn't do any premature optimizations until we have a reason for it.

> Also, rnpm already uses xml reader for android-manifest, so I think we could use it instead to avoid string parsing - https://github.com/rnpm/rnpm/blob/master/package.json#L47

Meh, we can do it, but I think it's +1 dependency and I don't see any benefits of having it there. Probably it'll remove my regexp parsing code, but what's the point?

> However - if we go with separate rnpm.xml that we fully manage, that should not be the issue since we create file, update it and delete when neccessary.

What's the problem with default strings.xml? You was against any rnpm-specific files, so why now? Which issues do you see using strings.xml?

> Also we can define other, non string parameters (that should be easier with xmldoc as I proposed in previous comment).

We don't have any use-cases so far. It that would be a case, I'll replace my implementation by `xmldoc`.